removeDuplicates.js

```diff
@@ -0,0 +1,12 @@
+const removeDuplicate = (str) =>{
+    let mystr = ""
+    for (let char of str){
+        if (mystr.includes(char)== false){
+            mystr += char
+        }
+    }
```

Comment:
Instead of using loops, you can use `Set`. Read up about `Set` and `Map` in javascript.

Something like this is better `[...new Set(value)].join('')`

Reply:
fixed on b00c151